src/client/components/equalizer/equalizer.js

```diff
@@ -189,8 +187,38 @@ class Equalizer extends Component {
       widthCanvas,
       heightCanvas,
       trackName,
+      sound,
+      startMuteState,
     } = this.props.audioData;
 
+    const hideOrShowCanvas = sound || startMuteState ? (
```

Comment:
No need to repeat code, you could use ternary on styles you want to change

Reply:
done